src/utils.ts

```diff
@@ -82,6 +82,12 @@ export type NavigationBarIcon = Readonly<{
      * These properties will be merged to the tracking event produced by the native side
      */
     trackingProperties?: Record<string, string>;
+
+    /**
+     * Identifies the action as opening a section subject to maintenance mode rules, and which section it is.
+     * If Non null it may be that the native app will show a maintenance overlay instead of the usual behaviour when clicked.
+     */
+    maintenanceSectionName?: string;
```

Comment:
If I understand this correctly this parameter is to give a hint to the native side of which section will be opened using this NavBar Icon.

I wouldn't couple this to the "maintenance" feature. This could just be named as `sectionName`. This hint could be used for maintenance or other needs.

Reply:
maintenanceSectionName was suggested by native team. we would need to align with them

Here's an example of the associated config: https://github.com/Telefonica/unified_config/blob/ba26458dc4f0109b0f862b82610d441733d55d60/main/common/appsMaintenanceModeConfig.yaml#L4

Consiering the rithm  by which we define new features, I don't see much value in adding to the doc a snapshot of our current state, as it's variable by OB and through time which sections of the app will exist (and will eventually all by in a CMS)

Reply:
Ok, then perhaps it would be useful a link to the config in the readme.

Reply:
How about a comment saying:

-  for NOVUM app you can check `appsMaintenanceModeConfig` to see which sections have been defined for maintenance mode purposes

I'm unsure if NOVUM app should be mentioned, as I'm unsure of the scope of the webview-bridge, maybe other apps like smart wifi share this bridge?

Reply:
This new value comes from the need of knowing from native when a toolbar button which launches a webview could be associated to a maintenance section already managed from native side.

If that section maintenance is enabled, instead of opening the link associated, we need to launch the associated maintenance native screen.

maintenanceSectionNames as in this config, can be considered ids for that --> https://github.com/Telefonica/unified_config/blob/ba26458dc4f0109b0f862b82610d441733d55d60/main/common/appsMaintenanceModeConfig.yaml#L4

Reply:
> I'm unsure if NOVUM app should be mentioned
Perhaps, instead of a comment in the README, this could be commented internally, as a source code comment